linera-execution/src/lib.rs

```diff
@@ -52,9 +42,20 @@ use linera_base::{
     ownership::ChainOwnership,
 };
 use linera_views::{batch::Batch, views::ViewError};
+pub use policy::ResourceControlPolicy;
+pub use resources::{ResourceController, ResourceTracker};
 use serde::{Deserialize, Serialize};
-use std::{fmt, str::FromStr, sync::Arc};
+pub use system::{
+    SystemExecutionError, SystemExecutionStateView, SystemMessage, SystemOperation, SystemQuery,
+    SystemResponse,
+};
 use thiserror::Error;
+#[cfg(all(with_testing, any(with_wasmer, with_wasmtime)))]
+pub use wasm::test as wasm_test;
+#[cfg(with_wasm_runtime)]
+pub use wasm::{WasmContractModule, WasmExecutionError, WasmServiceModule};
```

Comment:
Nit: my bad, we should have probably prefixed these with `self::`

```suggestion
pub use self::wasm::test as wasm_test;
#[cfg(with_wasm_runtime)]
pub use self::wasm::{WasmContractModule, WasmExecutionError, WasmServiceModule};
```

Reply:
Bearing in mind that this PR is going to get squashed into a single commit, I'm really keen to keep this commit just the application of `group_imports`, because `git` will generate a ton of merge conflicts with it that can be fixed just by running `cargo fmt` — unless we start folding more changes into the commit.